scripts/integtest.sh

```diff
@@ -8,48 +8,67 @@ function usage() {
     echo "--------------------------------------------------------------------------"
     echo "Usage: $0 [args]"
     echo ""
-    echo "Required arguments:"
-    echo "None"
-    echo ""
     echo "Optional arguments:"
-    echo -e "-b BIND_ADDRESS\t, defaults to localhost | 127.0.0.1, can be changed to any IP or domain name for the cluster location."
-    echo -e "-p BIND_PORT\t, defaults to 9200, can be changed to any port for the cluster location."
-    echo -e "-t TRANSPORT_PORT\t, defaults to 9300, can be changed to any port for the cluster location."
     echo -e "-s SECURITY_ENABLED\t(true | false), defaults to true. Specify the OpenSearch/Dashboards have security enabled or not."
     echo -e "-c CREDENTIAL\t(usename:password), no defaults, effective when SECURITY_ENABLED=true."
-    echo -e "-h\tPrint this message."
+    echo -e "-h Print this message."
     echo -e "-v OPENSEARCH_VERSION\t, no defaults"
-    echo -e "-n SNAPSHOT\t, defaults to false"
+    echo -e "-n SNAPSHOT\t\t, defaults to false"
+    echo "Required arguments:"
+    echo "Single cluster test:"
+    echo -e "-b BIND_ADDRESS\t\t, IP or domain name for the cluster location."
+    echo -e "-p BIND_PORT\t\t, port for the cluster location."
+    echo -e "-t TRANSPORT_PORT\t, defaults to 9300, can be changed to any port for the cluster location."
+    echo "--------------------------------------------------------------------------"
+    echo "Multi cluster test:"
+    echo -e "-m Leader BIND_ADDRESS\t\t, defaults to localhost | 127.0.0.1, can be changed to any IP or domain name for the cluster location."
```

Comment:
Lets decide on generic params here which can be extended by other plugins as well.